codecov_auth/migrations/0036_add_user_terms_agreement.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.2 on 2023-08-30 13:27
+
+from django.db import migrations, models
+
+import core.models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
Could you add the sqlmigrate output here? `python manage.py sqlmigrate codecov_auth 0036`? This helps us know the sql for this command, we usually try to put it in migration files